studio/rabbit_queue.py

```diff
@@ -383,13 +390,13 @@ def close_channel(self):
         Close channel by sending the Channel.Close RPC command.
         """
         with self._rmq_lock:
-            if self._channel is not None:
+            if self._channel_is_valid():
```

Comment:
pika library is really upset if we try to close already closed object,
so put in guard checks.